v4.14/graph.yaml

```diff
@@ -12,7 +12,9 @@ entries:
   replaces: orchestrator-operator.v1.2.0
 - name: orchestrator-operator.v1.2.3
   replaces: orchestrator-operator.v1.2.1
-name: stable-1.2
+- name: orchestrator-operator.v1.3.0
```

Comment:
and I think you might be missing adding this as well:
`skipRange: '>=1.2.3 <1.3.0'`

Reply:
Updated this

Reply:
I have a doubt about this because we are keeping each release in its own channel. If we had a `stable` channel that contains all releases, we would need this. But if we keep 1.2 and 1.3 releases in their own channels, then I think we don't need these expressions. But the downside is that customers will need to manually change the channel, and I don't know if that can be done over an installed environment. @pkliczewski @masayag what do you think if we reuse the `stable-1.2` for 1.3, 1.4, etc... as well and rename it to `stable`?